blueocean-core-js/src/js/services/ActivityService.js

```diff
@@ -90,6 +90,9 @@ export class ActivityService extends BunkerService {
                     run.result = 'UNKNOWN';
                 }
                 return this.setItem(run);
+            })
+            .catch(err => {
+                console.log('There had been an error while trying to get the data.', err); // FIXME: Ivan what is the way to return an "error" opbject so underlying component are aware of the problem and can react
```

Comment:
I talked to @imeredith and he wanted to extend the pager to return an object such as:

```
{ data, error, state }
```

That will hopefully go away anyway but will fix the working.